src/Pages/MainPage/Components/TaskForm.tsx

```diff
@@ -9,9 +9,13 @@ import "./styles.css";
 
 interface TaskFormProps {
   addTask: (task: Task) => void;
+  deleteCompletedTask: () => void;
```

Comment:
```suggestion
  deleteCompletedTasks: () => void;
```
[NIT] as we are deleting multiple tasks here, this should be plural.

Always best to be very precise about singular or plural when naming.